specs/binding-mqtt-kafka.spec/src/main/scripts/io/aklivity/zilla/specs/binding/mqtt/kafka/streams/kafka/publish.one.message.changed.topic.name/client.rpt

```diff
@@ -39,7 +39,7 @@ write zilla:data.ext ${kafka:dataEx()
                                .header("zilla:filter", "sensor")
                                .header("zilla:filter", "one")
                                .header("zilla:local", "client")
-                               .headerInt("zilla:timeout-ms", 15000)
+                               .headerInt("zilla:expiry", 15000)
```

Comment:
The unit should also be adjusted to `seconds`, not `milliseconds`, per previous feedback please.

See https://docs.oasis-open.org/mqtt/mqtt/v5.0/os/mqtt-v5.0-os.html#_Toc3901112